Automattic/woocommerce-payments
# feat: add saved cards setting

Fixes https://github.com/Automattic/woocommerce-payments/issues/2417

#### Changes proposed in this Pull Request









Adding the "Saved cards" option to the settings.

![2021-07-16 14 28 09](https://user-images.githubusercontent.com/273592/125999227-671b0134-043f-4c54-a78d-252805564f02.gif)


#### Testing instructions





- Ensure you have the "Enable grouped settings" flag enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments
- The "Enable payments via saved cards" checkbox is present under the "Transaction preferences" section
- The checkbox's status is persisted on page refresh
- Updating the value updates the (pre-existing) `saved_cards` option in the gateway

-------------------
